test/forms/declaration/DeclarationHolderSpec.scala

```diff
@@ -250,53 +141,10 @@ class DeclarationHolderSpec extends WordSpec with MustMatchers with JourneyTypeT
     }
   }
 
-  "DeclarationHolder on form method" when {
-
-    onSimplified { implicit request =>
-      "provided with empty holders list" should {
-        "return mandatoryMapping" in {
-
-          val holders = Seq.empty[DeclarationHolder]
-
-          val result = DeclarationHolder.form(holders)
-
-          result.mapping mustBe DeclarationHolder.mandatoryMapping
-        }
-      }
-
-      "provided with non-empty holders list" should {
-        "return mandatoryMapping" in {
-
-          val holders = Seq(ExportDeclarationTestData.correctDeclarationHolder)
-
-          val result = DeclarationHolder.form(holders)
-
-          result.mapping mustBe DeclarationHolder.mandatoryMapping
-        }
-      }
-    }
-
-    onJourney(STANDARD, SUPPLEMENTARY, OCCASIONAL, CLEARANCE) { implicit request =>
-      "provided with empty holders list" should {
-        "return optionalMapping" in {
-
-          val holders = Seq.empty[DeclarationHolder]
-
-          val result = DeclarationHolder.form(holders)
-
-          result.mapping mustBe DeclarationHolder.optionalMapping
-        }
-      }
-
-      "provided with non-empty holders list" should {
-        "return mandatoryMapping" in {
-
-          val holders = Seq(ExportDeclarationTestData.correctDeclarationHolder)
-
-          val result = DeclarationHolder.form(holders)
-
-          result.mapping mustBe DeclarationHolder.mandatoryMapping
-        }
+  "DeclarationHolder on form method" should {
+    onEveryDeclarationJourney() { implicit request =>
```

Comment:
Once you remove the request parameter from this method, then testing it for every journey is redundant.